intermediate_source/dist_tuto.rst

```diff
@@ -2,6 +2,8 @@ Writing Distributed Applications with PyTorch
 =============================================
 **Author**: `Séb Arnold <https://seba1511.com>`_
 
+**Edited by**: `Chirag Pandya <https://github.com/c-p-i-o>`_
+
```

Comment:
We typically don't add these type of lines.
```suggestion
```

Reply:
Cool! Didn't know that. The last tutorial I worked on had the `Edited by` line.